vars/simpleNode.groovy

```diff
@@ -18,7 +18,7 @@ def call(String label, int timeoutInMinutes, Closure body) {
     node (label) {
         // Clean.  Currently processes are killed at the end of the node block, but we don't have an easy way to run the cleanup
         // after the node block exits currently.  Cleaning at the start should be sufficient.
```

Comment:
Remove `currently` at the end of the sentence - `...after the node block exits currently`. 
Groovy novice. So learning through comments.